pytorch_lightning/metrics/classification/helpers.py

```diff
@@ -364,7 +371,8 @@ def _input_format_classification(
         target: Tensor with ground truth labels, always integers (labels)
         threshold:
             Threshold probability value for transforming probability predictions to binary
-            (0,1) predictions, in the case of binary or multi-label inputs. Default: 0.5
+            (0 or 1) predictions, in the case of binary or multi-label inputs. If not set it
+            defaults to 0.5.
         num_classes:
             Number of classes. If not explicitly set, the number of classes will be infered
             either from the shape of inputs, or the maximum label in the ``target`` and ``preds``
```

Comment:
same here few lines below, should is_multiclass does be updated here too?? using:
```
:ref:`documentation section <metrics:Using the \\`\\`is_multiclass\\`\\` parameter>`
```
```

Reply:
mind check other places too if required

Reply:
Good catch